python/python/glide/async_commands/cluster_commands.py

```diff
@@ -68,6 +68,8 @@ async def info(
             information to retrieve. When no parameter is provided, the default option is assumed.
             route (Optional[Route]): The command will be routed to all primaries, unless `route` is provided, in which
             case the client will route the command to the nodes defined by `route`. Defaults to None.
+            When `route` is set to `allNodes`, the `INFO` command may return partial data, as the response from each node
+            might be incomplete due to factors like network conditions or node load. This can lead to varying results across different executions.
```

Comment:
please add it to the java client as well. 
Do we want this comment to all Special policy commands? @barshaul